### ISSUE TITLE
Photo Album features

### ISSUE THREAD

**Issue by imanolGOMI** [sudharti/socify#26]
I have seen in other branch that the feature is implemented but in master not appear. ¿Why?

**Comment by scaffeinate** [comment]
@w0www Yes it is still not stable yet. I need to fix a few issues before merging it back to master. And since this links to a blog post tutorial I am planning to create a separate branch for that and then merge everything into master 👍

**Comment by imanolGOMI** [comment]
@sudharti  Ok!! thanks. I am advancing in a fork. You can see the changes my master branch. I have photoAlbum but get some errors.

**Comment by scaffeinate** [comment]
@w0www Could you tell me more info on the error you are facing?

**Comment by imanolGOMI** [comment]
Now appear this error:
When i start the app i see:
`ActionController::UrlGenerationError at /`
`No route matches {:action=>"mentionable", :controller=>"users", :id=>nil} missing required keys: [:id]`

in app/views/comments/_form.html.erb

But before when i had created the PhotoAlbum appeared an error that said "not commentable method for Class"

**Comment by imanolGOMI** [comment]
![seleccion_001](https://cloud.githubusercontent.com/assets/1654762/19942723/ff41fae2-a134-11e6-9be8-71ad10e2f23f.png)

**Comment by scaffeinate** [comment]
@w0www From your Pull Request https://github.com/w0www/socify/pull/2 I see you have merged develop into master. Develop has a lot of new features and refactoring. So you cannot automatically merge it. Some files need to be reviewed. Didn't you get any conflicts while merging? Or if you had rebased you would have got a few merge conflicts.

**Comment by scaffeinate** [comment]
The problem you are facing is because the develop branch doesn't have the mentionable feature yet. If you see https://github.com/sudharti/socify/blob/develop/app/views/comments/_comment.html.erb and the one in master https://github.com/sudharti/socify/blob/master/app/views/comments/_form.html.erb. You see the master one has user mentionable feature while develop doesn't. I will work on merging it onto master. But if you want to resolve it you need to update it to the latest in develop.

**Comment by imanolGOMI** [comment]
@sudharti  yes i get a lot of conflicts while merging. 

First i have tried merge Master into develop to get the last changes of master. Then i have done a PR from develop to Master without conflicts.

I may be wrong when i do merge because I not know the changes

**Comment by scaffeinate** [comment]
@w0www Sorry about that. I did receive a few Pull Requests to master while I was working on the develop branch. So kind of messed up a little there. But I will be merging develop into master once it's stable. I will keep this issue open. Will notify you if you want.

**Comment by imanolGOMI** [comment]
Yes, please notify me. And if you has more ideas or more features, i can help you, you know! ;)

Thanks mate!

**Comment by scaffeinate** [comment]
@w0www Cool. Thanks 👍
